Sources/Storyboard/StoryboardSceneBased.swift

```diff
@@ -42,8 +42,8 @@ public extension StoryboardSceneBased where Self: UIViewController {
    - returns: instance of the conforming ViewController
    */
   static func instantiate() -> Self {
-    let storyboard = Self().storyboard
-    guard let vc = storyboard?.instantiateViewController(withIdentifier: self.sceneIdentifier) as? Self else {
```

Comment:
_(To me those two lines after the only one needing to be changed, to fix the bug you mention)_